shared/common-adapters/back-button.native.js

```diff
@@ -30,11 +30,6 @@ export default class BackButton extends Component {
   }
 }
 
-BackButton.propTypes = {
```

Comment:
we're using flow instead of prop types and they moved that to a different repo out of react